### ISSUE TITLE
PDFJS Table of Contents Contrast Issues

### ISSUE THREAD

**Issue by StevenDoesStuffs** [James-Yu/LaTeX-Workshop#2892]
### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes
### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: N/A

## Describe the bug [Required]

The table of contents in the PDF viewer has a dark font on a dark background, exactly as described in #2622. 
That issue has been marked as a duplicate of a fixed issue, but the problem still persists.

### To Reproduce

1. Open a PDF
2. Show table of contents
3. See error

### Expected behavior

Either the text should be light or the background should be light. 

## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

### LaTeX Workshop Output [Required]

https://pastebin.com/Fgbgus3w

### Developer Tools Console [Required]

https://pastebin.com/XLUHEMX6

## Screenshots

![Screenshot_20211006_030751](https://user-images.githubusercontent.com/13265529/136164499-cb117137-d7e1-4947-88aa-c4cd945d6ea7.png)

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: Arch Linux
 - VS Code version: 1.60.2
 - Extension version: 8.20.2
 - TeX distribution version: TeX 3.141592653 (TeX Live 2021/Arch Linux)

### Are you using VSCodium?

No. I specifically installed the [official binary distribution](https://aur.archlinux.org/packages/visual-studio-code-bin/) to reproduce this issue.

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

### Additional context

I am using a dark GTK theme.

**Comment by tamuratak** [comment]
```
[02:58:57] Initializing LaTeX Workshop.
[02:58:57] Extension root: /home/stevendoesstuffs/.vscode/extensions/james-yu.latex-workshop-8.20.2
[02:58:57] $PATH: /home/stevendoesstuffs/.yarn/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/stevendoesstuffs/.antigen/bundles/robbyrussell/oh-my-zsh/lib:/home/stevendoesstuffs/.antigen/bundles/zsh-users/zsh-syntax-highlighting
[02:58:57] $SHELL: /bin/zsh
[02:58:57] vscode.env.appName: Visual Studio Code
[02:58:57] vscode.env.remoteName: undefined
[02:58:57] vscode.env.uiKind: 1
[02:58:57] latex-workshop.bind.enter.key: true
[02:58:57] latex-workshop.docker.enabled: false
[02:58:57] latex-workshop.docker.image.latex: ""
[02:58:57] latex-workshop.intellisense.package.enabled: true
[02:58:57] latex-workshop.intellisense.update.aggressive.enabled: false
[02:58:57] latex-workshop.intellisense.update.delay: 1000
[02:58:57] latex-workshop.latex.autoBuild.run: "onFileChange"
[02:58:57] latex-workshop.latex.outDir: "%DIR%"
[02:58:57] latex-workshop.latex.recipes: [
 {
  "name": "latexmk 🔃",
  "tools": [
   "latexmk"
  ]
 },
 {
  "name": "latexmk (latexmkrc)",
  "tools": [
   "latexmk_rconly"
  ]
 },
 {
  "name": "latexmk (lualatex)",
  "tools": [
   "lualatexmk"
  ]
 },
 {
  "name": "pdflatex ➞ bibtex ➞ pdflatex × 2",
  "tools": [
   "pdflatex",
   "bibtex",
   "pdflatex",
   "pdflatex"
  ]
 },
 {
  "name": "Compile Rnw files",
  "tools": [
   "rnw2tex",
   "latexmk"
  ]
 },
 {
  "name": "Compile Jnw files",
  "tools": [
   "jnw2tex",
   "latexmk"
  ]
 },
 {
  "name": "tectonic",
  "tools": [
   "tectonic"
  ]
 }
]
[02:58:57] latex-workshop.latex.tools: [
 {
  "name": "latexmk",
  "command": "latexmk",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "-pdf",
   "-outdir=%OUTDIR%",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "lualatexmk",
  "command": "latexmk",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "-lualatex",
   "-outdir=%OUTDIR%",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "latexmk_rconly",
  "command": "latexmk",
  "args": [
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "pdflatex",
  "command": "pdflatex",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "bibtex",
  "command": "bibtex",
  "args": [
   "%DOCFILE%"
  ],
  "env": {}
 },
 {
  "name": "rnw2tex",
  "command": "Rscript",
  "args": [
   "-e",
   "knitr::opts_knit$set(concordance = TRUE); knitr::knit('%DOCFILE_EXT%')"
  ],
  "env": {}
 },
 {
  "name": "jnw2tex",
  "command": "julia",
  "args": [
   "-e",
   "using Weave; weave(\"%DOC_EXT%\", doctype=\"tex\")"
  ],
  "env": {}
 },
 {
  "name": "jnw2texmintex",
  "command": "julia",
  "args": [
   "-e",
   "using Weave; weave(\"%DOC_EXT%\", doctype=\"texminted\")"
  ],
  "env": {}
 },
 {
  "name": "tectonic",
  "command": "tectonic",
  "args": [
   "--synctex",
   "--keep-logs",
   "%DOC%.tex"
  ],
  "env": {}
 }
]
[02:58:57] latex-workshop.viewer.pdf.internal.keyboardEvent: "auto"
[02:58:57] Creating a new file watcher.
[02:58:57] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{"stabilityThreshold":250}}
[02:58:57] Creating PDF file watcher.
[02:58:57] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{}}
[02:58:57] Creating Bib file watcher.
[02:58:57] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{"stabilityThreshold":250}}
[02:58:57] Set $LATEXWORKSHOP_DOCKER_LATEX: ""
[02:58:57] [Server] Creating LaTeX Workshop http and websocket server.
[02:58:57] LaTeX Workshop initialized.
[02:58:57] Trigger characters for intellisense of LaTeX documents: ["\\",",","{"]
[02:58:57] Bibtex format config: {"tab":"  ","case":"lowercase","left":"{","right":"}","trailingComma":false,"sort":["key"],"alignOnEqual":true,"sortFields":false,"fieldsOrder":[]}
[02:58:57] Current workspace folders: undefined
[02:58:57] Current workspaceRootDir: 
[02:58:57] [Server] Server successfully started: {"address":"127.0.0.1","family":"IPv4","port":42123}
[02:58:57] Snippet data loaded.
[02:58:58] Added to PDF file watcher: /home/stevendoesstuffs/Important/Books/qcqi.pdf
[02:58:59] PDF command invoked.
[02:58:59] Serving PDF file at http://127.0.0.1:42123/viewer.html?file=pdf..JTJGaG9tZSUyRnN0ZXZlbmRvZXNzdHVmZnMlMkZJbXBvcnRhbnQlMkZCb29rcyUyRnFjcWkucGRm
[02:58:59] The encoded path is /home/stevendoesstuffs/Important/Books/qcqi.pdf
[02:58:59] The internal PDF viewer url: http://127.0.0.1:42123/viewer.html?incode=1&file=pdf..JTJGaG9tZSUyRnN0ZXZlbmRvZXNzdHVmZnMlMkZJbXBvcnRhbnQlMkZCb29rcyUyRnFjcWkucGRm
[02:58:59] Open PDF tab for /home/stevendoesstuffs/Important/Books/qcqi.pdf
[02:58:59] Handle data type: open
[02:58:59] Preview PDF file: /home/stevendoesstuffs/Important/Books/qcqi.pdf
[02:58:59] Handle data type: request_params
[02:58:59] Sending the settings of the PDF viewer for initialization: {"type":"params","scale":"auto","trim":0,"scrollMode":0,"spreadMode":0,"hand":false,"invertMode":{"enabled":true,"brightness":1.25,"grayscale":0.6,"hueRotate":180,"invert":0.9,"sepia":0},"bgColor":"#ffffff","keybindings":{"synctex":"ctrl-click"}}
[02:58:59] Handle data type: loaded
```

```
workbench.desktop.main.js:sourcemap:71 [Extension Host] (node:79219) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.(Use `code --trace-deprecation ...` to show where the warning was created)
v @ workbench.desktop.main.js:sourcemap:71
viewer.js:1581 PDF 4dd0dfa3f465ca8e8ee6874fd1030a0e [1.6 - / -] (PDF.js: 2.8.335)
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/out/viewer/latexworkshop.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/out/viewer/components/connection.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/out/viewer/components/synctex.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/out/viewer/components/utils.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/out/viewer/components/pagetrimmer.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/out/viewer/components/viewerhistory.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/build/pdf.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/viewer.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for http://127.0.0.1:42123/build/pdf.worker.js.map: Fetch through target failed: Target not supported; Fallback: Unknown error
```

**Comment by tamuratak** [comment]
Fixed in 6d4f1dd04e8621c5528b0e0813f556e44f4dc27d

**Comment by tamuratak** [comment]
Please don't use https://pastebin.com next time.
